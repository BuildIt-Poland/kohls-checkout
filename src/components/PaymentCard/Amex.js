import React from 'react';
import PropTypes from 'prop-types';

import Wrapper from './Wrapper';
import { AmexCard } from '../Icons';

function Amex({ small }) {
  return (
    <Wrapper small={small}>
      <AmexCard />
    </Wrapper>
  );
}

Amex.propTypes = {
  cardNumber: PropTypes.bool
};

export default Amex;
