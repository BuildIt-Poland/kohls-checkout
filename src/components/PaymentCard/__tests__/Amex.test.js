import React from 'react';
import { create } from 'react-test-renderer';

import Amex from '../Amex';

describe('COMPONENT - Amex', () => {
  it('renders Amex component', () => {
    const component = create(<Amex />);

    expect(component.toJSON()).toMatchSnapshot();
  });

  it('renders small Amex component', () => {
    const component = create(<Amex small />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
