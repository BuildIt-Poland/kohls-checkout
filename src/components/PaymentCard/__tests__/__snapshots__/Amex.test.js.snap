// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Amex renders Amex component 1`] = `
.c0 {
  width: 132px;
}

<div
  className="c0"
>
  <MasterCardIcon />
</div>
`;

exports[`COMPONENT - Amex renders small Amex component 1`] = `
.c0 {
  width: 40px;
}

<div
  className="c0"
>
  <MasterCardIcon />
</div>
`;
